draft-ietf-rats-msg-wrap.md

```diff
@@ -319,6 +319,28 @@ with the following wire representation:
    03                                 # unsigned(3)
 ~~~
 
+# CMW Collections {#cmw-coll}
+
+Composite and layered attesters produce evidence made of multiple evidence "bits".
```

Comment:
I don't especially like the colloquialism.

Reply:
see [ac1bfe3](https://github.com/ietf-rats-wg/draft-ietf-rats-msg-wrap/pull/42/commits/ac1bfe37b332e70249093b959ff8dc34a153a520)